regression/cbmc-java/generic_class_bound1/Gn.java

```diff
@@ -0,0 +1,15 @@
+interface A{}
```

Comment:
It might be better to structure this as a unit test (consider checking/rebasing onto #1534) to ensure the parsing is happening as expected.